cluster/ubuntu/build.sh

```diff
@@ -54,10 +53,10 @@ cp $ETCD/etcd $ETCD/etcdctl binaries/master
 
 # k8s
 echo "Download kubernetes release ..."
-K8S_VERSION=${K8S_VERSION:-"1.0.3"}
+KUBE_VERSION=${KUBE_VERSION:-"1.0.3"}
```

Comment:
Or we can use `K8S_VERSION=$KUBE_VERSION` in prepare-push func, which seems a little messy to me.